src/tests/intg/test_session_recording.py

```diff
@@ -287,31 +287,54 @@ def all(request, ldap_conn, users_and_groups):
 def test_all_nam(all):
     """Test "all" scope with getpwnam"""
     ent.assert_each_passwd_by_name(dict(
-        user1=dict(name="user1", uid=1001, shell=config.SESSION_RECORDING_SHELL),
-        user2=dict(name="user2", uid=1002, shell=config.SESSION_RECORDING_SHELL),
-        user3=dict(name="user3", uid=1003, shell=config.SESSION_RECORDING_SHELL),
-        user4=dict(name="user4", uid=1004, shell=config.SESSION_RECORDING_SHELL),
+        user1=dict(name="user1",
+                   uid=1001,
+                   shell=config.SESSION_RECORDING_SHELL),
+        user2=dict(name="user2",
+                   uid=1002,
+                   shell=config.SESSION_RECORDING_SHELL),
+        user3=dict(name="user3",
+                   uid=1003,
+                   shell=config.SESSION_RECORDING_SHELL),
+        user4=dict(name="user4",
+                   uid=1004,
+                   shell=config.SESSION_RECORDING_SHELL),
     ))
```

Comment:
Following diff works pass pep8 as well and require less new lines.
@@ -287,20 +287,28 @@ def all(request, ldap_conn, users_and_groups):
 def test_all_nam(all):
     """Test "all" scope with getpwnam"""
     ent.assert_each_passwd_by_name(dict(
-        user1=dict(name="user1", uid=1001, shell=config.SESSION_RECORDING_SHELL),
-        user2=dict(name="user2", uid=1002, shell=config.SESSION_RECORDING_SHELL),
-        user3=dict(name="user3", uid=1003, shell=config.SESSION_RECORDING_SHELL),
-        user4=dict(name="user4", uid=1004, shell=config.SESSION_RECORDING_SHELL),
+        user1=dict(name="user1", uid=1001,
+                   shell=config.SESSION_RECORDING_SHELL),
+        user2=dict(name="user2", uid=1002,
+                   shell=config.SESSION_RECORDING_SHELL),
+        user3=dict(name="user3", uid=1003,
+                   shell=config.SESSION_RECORDING_SHELL),
+        user4=dict(name="user4", uid=1004,
+                   shell=config.SESSION_RECORDING_SHELL),
     ))
```

```